### ISSUE TITLE
Remove personal information from the dashboard

### ISSUE THREAD

**Issue by babrubhan** [Manvityagi/Entry-Management-Software#3]
Great work but data privacy is important!! https://hidden-savannah-59110.herokuapp.com/dashboard

We can put dummy data for testing ..otherwise it should be encrypted.

**Comment by Manvityagi** [comment]
Yeah, You are right @babrubhan, will delete the data.  Most of the data is just dummy though.
